src/System.Management.Automation/resources/RemotingErrorIdStrings.resx

```diff
@@ -1636,7 +1636,10 @@ All WinRM sessions connected to Windows PowerShell session configurations, such
   <data name="InvalidRoleCapabilityFileExtension" xml:space="preserve">
     <value>The provided role capability file {0} does not have the required .psrc extension.</value>
   </data>
-  <data name="SSHAbruptlyTerminated" >
+  <data name="SSHAbruptlyTerminated" xml:space="preserve">
     <value>The SSH transport process has abruptly terminated causing this remote session to break.</value>
   </data>
+  <data name="WSManClientDllNotAvailable" xml:space="preserve">
+    <value>WSMan client library is not installed</value>
```

Comment:
@joeyaiello I'm fine with everything except the last sentence as SSH may or may not be installed.  I don't think we need to go to that much detail at this point.  Ideally, we should just tell the user `run Enable-PSRemoting` and magic happens, but that's not anytime soon.